samples/esb/esp_ptx/README.rst

```diff
@@ -37,12 +16,20 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
+Additionally, if you want to test the Enhanced ShockBurst Receiver functionality, you need to build and run the :ref:`esb_prx` sample.
 You can use any two of the development kits listed above and mix different development kits.
 
-Configuration
-*************
+Overview
+********
 
-|config|
+The sample consists of a Transmitter that use the :ref:`esb_README` library.
+After programming each application on a nRF52 Series development kit, you can test that packets that are sent by the kit that runs the Transmitter application are picked up by the kit that runs the Receiver application (by additionally building and running the :ref:`esb_prx` sample.
+Successful communication is indicated by LED changes, which should be in sync on both kits.
+
+The Transmitter sends a packet, waits for a configurable time (50 milliseconds by default), and then sends another packet.
+In each packet, the four least significant bits of the first byte of the payload are incremented (or reset to zero when they reach 16).
```

Comment:
I don't really get how this explains the pattern as described in the User interface section. Especially where the 16 comes from.

Reply:
Good suggestion - I will look more into this. I believe GF has some concerns about this, although that is how it was done for the Gazell samples (which were also split for the same reasons/in a similar way).

Reply:
@grochu Could you please advise on this - is there a clearer way to explain this perhaps?

Reply:
@grochu Please let me know what you think of this; if anything can be added to improve clarity here.

Reply:
It's the 4 bits that are incremented and the maximum value represented by 4 bits is 15, so the 16th packet will have these bits zeroed and incrementing goes up again in the next packets. Maybe we could state something like:
```suggestion
In each packet, the four least significant bits of the first byte of the payload are incremented (the transmitted number will count up from 0 to 15 and wrap around).
```

Reply:
Yeah, basically the LES pattern from the User Interface section is executed twice on each block of 16 packets (numbers 0 - 15). But essentially these 2 things are independent: if the number transmitted over the air used 5 bits or more, the LEDs would behave the same - would light up 1-2-3-4 and go off 1-2-3-4 on every block of 8 packets - they use modulo counting. So I think both statements are correct.

Reply:
How important is the information about incrementing the bits? I feel the necessary information is conveyed even without this sentence.

Reply:
We may remove the sentence I guess, it can be treated as an implementation detail. What the user sees are the LEDs.

Reply:
Removed, and other suggestion applied in new PR here: https://github.com/nrfconnect/sdk-nrf/pull/12362